docs/api/connectivity/bluetooth/BLE.md

```diff
@@ -1,76 +1,92 @@
 
-Bluetooth low energy (BLE) is a low power wireless technology standard for building personal area networks. Typical applications of BLE are health care, fitness trackers, beacons, smart home, security, entertainment, proximity sensors, industrial and automotive.
+Arm Mbed BLE is the Bluetooth Low Energy software solution for Mbed. Many Mbed
+[targets and components](https://os.mbed.com/platforms/?mbed-enabled=15&connectivity=3) support Mbed BLE. Developers can
+use it to create new BLE enabled applications.
 
-Arm Mbed BLE, also called `BLE_API`, is the Bluetooth Low Energy software solution for Mbed. Many Mbed [targets and components](https://os.mbed.com/platforms/?mbed-enabled=15&connectivity=3) support Mbed BLE. Developers can use it to create new BLE enabled applications.
+## BLE API
 
-Mbed’s `BLE_API` interfaces with the BLE controller on the board. It hides the BLE stack’s complexity behind C++ abstractions and is compatible with all BLE-enabled Mbed board. The Mbed OS `BLE_API` automatically configuring the clocks, timers and other hardware peripherals to work at their lowest power consumption.
+Mbed's BLE API is available through C++ classes. It hides the BLE stack’s complexity and is compatible with all
+BLE-enabled Mbed board. It automatically configures the clocks, timers and other hardware peripherals to work at their
+lowest power consumption.
 
-## `BLE_API`, bridges and stacks
+### BLE API headers
 
-<span class="images">![](../../../images/BLEDiagram.png)</span>
+BLE API is accessible through several header files:
 
-You can build a BLE application using Mbed OS, `BLE_API` and a controller-specific Bluetooth stack together with some bridge software to adapt it to `BLE_API`:
+- [BLE.h](https://github.com/ARMmbed/mbed-os/blob/master/connectivity/FEATURE_BLE/include/ble/BLE.h) - acquire the BLE
+  instance, perform initialisation
+- [Gap.h](https://github.com/ARMmbed/mbed-os/blob/master/connectivity/FEATURE_BLE/include/ble/Gap.h) - advertising,
+  scanning, connecting
+- [GattClient.h](https://github.com/ARMmbed/mbed-os/blob/master/connectivity/FEATURE_BLE/include/ble/GattClient.h) -
+  GATT operations as client
+- [GattServer.h](https://github.com/ARMmbed/mbed-os/blob/master/connectivity/FEATURE_BLE/include/ble/GattServer.h) -
+  GATT operations as server
+- [SecurityManager.h](https://github.com/ARMmbed/mbed-os/blob/master/connectivity/FEATURE_BLE/include/ble/SecurityManager.h) -
+  authentication, keys, encryption
 
-- `BLE_API` as described above.
-- The bridge software is specific to each vendor’s board. It provides the instantiations for the interfaces `BLE_API` offers and helps drive the underlying controller and Bluetooth stack.
-- The Bluetooth stack implements the Bluetooth protocol and is specific to the controller, so a vendor using different controllers may provide different stacks.
+Specific documentation for each component is available inside each of these headers.
 
-## Inside `BLE_API`
+### BLE stacks
```

Comment:
I think this section can go at the bottom, that's important but maybe not the most useful information for common users/